src/screens/LockedWallet.js

```diff
@@ -120,16 +120,27 @@ class LockedWallet extends React.Component {
    */
   handleReset = () => {
     $('#confirmResetModal').modal('hide');
+    // Remove from list of wallets
+    hathorLib.storage.store.removeWallet(hathorLib.storage.store.prefix);
+    // reset data
     wallet.resetWalletData();
-    this.props.history.push('/welcome/');
+    // If there are other wallets, go to screen to choose wallet
+    const firstWallet = wallet.getFirstWalletPrefix();
+    if (firstWallet) {
+      wallet.setWalletPrefix(firstWallet);
+      this.props.history.push('/choose_wallet');
+    } else {
+      wallet.setWalletPrefix(null);
+      this.props.history.push('/');
+    }
```

Comment:
Is this screen relevant?
We are adding more information to this screen but the "lock" now goes to the choose_wallet screen?
If it's not used anymore, can we remove it?

Reply:
As I understand, this might be used with the wallet service. The user might be required to input his pin from time to time.